### ISSUE TITLE
days and hour endpoints break on negative timezone difference than UTC

### ISSUE THREAD

**Issue by ijunaid8989** [evercam/evercam-server#1021]
It's mostly happening with the cameras which are on such timezones where its backwards from UTC.

I have discussed with @azharmalik3  This needs a deep thinking and explanation of the whole procedure how we are dealing with timezones, we see some mistakes in Timezone days and hours conversions, which are basically solutions for UTC+ timezones. 

I have tried to fix it but its breaking other timezones. So it needs a clear vision how to deal all timezones, not the default ones like Dublin (or any -UTC)

**Comment by marcoherbst** [comment]
I imagine this commentary from Milos is worth reading:
https://github.com/evercam/evercam-server/issues/353

M

**Comment by ijunaid8989** [comment]
The main issue was actually in days endpoint,

While doing hours, we were passing the UTC hour coming from Seaweedfs to shift it to the given timezone. but on days conversion we were passing time as 00:00:00 due to which each day was going -1. and disturbing the full calendar.

Fix: 

Instead of passing 00:00:00, I used current UTC time and pass the time to the year, month and day, so it will get converted from that time to any timezone not from 00:00:00. this way all -ve timezone are working now, and +ve as well, I have tested it the very last we have in time zones, International Date Line west, its working fine now.
